examples/meanstack/meanstack.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.1-dev
+
+services:
+- name: mongo
+  containers:
+  - image: mongo
+    ports:
+    - port: 27017:27017
+      type: external
+    mounts:
+    - volumeRef: mongo-persistent-storage
+      mountPath: /data/db
+
+- name: web
+  replicas: 2
+  containers:
+  - image: surajnarwade/myapp
+    ports: 
+    - port: 3000:80
+      type: external
+
+volumes:
+- name: mongo-persistent-storage
+  size: 100Mi
+  accessMode: ReadWriteMany
+  storageClass: fast
```

Comment:
`storageClass` should not be specified here.
If you remove this it will work on minikube without need to create special PV. And it will be also more universal so its possible to use it on other clusters.